src/features/filter-transactions/model.ts

```diff
@@ -1,6 +1,6 @@
-export type Status = 'committed' | 'rejected' | null;
-export type TabDefaultScreen = 'all' | 'transfers' | 'mints' | 'burns' | 'grants' | 'revokes';
-export type TabBlocksScreen = 'transactions' | 'transfers' | 'mints' | 'burns' | 'grants' | 'revokes';
+export type Status = 'Committed' | 'Rejected' | null;
```

Comment:
Remove this `Status` and use `TransactionStatus` from `schemas`. (or you can alias it, but not redefine)